flang/lib/Lower/Allocatable.cpp

```diff
@@ -935,6 +935,14 @@ Fortran::lower::genMutableBoxRead(Fortran::lower::FirOpBuilder &builder,
   return fir::AbstractBox{addr};
 }
 
+mlir::Value Fortran::lower::genIsAllocatedOrAssociatedTest(Fortran::lower::FirOpBuilder &builder, mlir::Location loc,                                const fir::MutableBoxValue &box) {
```

Comment:
Nit: something squirrely with this line happened.

Reply:
Fixed, thanks.